content/docs/command-reference/remote/modify.md

```diff
@@ -711,6 +711,12 @@ Read more about by expanding the WebHDFS section in
 > The user name may contain sensitive user info. Therefore, it's safer to add it
 > with the `--local` option, so it's written to a Git-ignored config file.
 
+- `kerb_ticket` - path to Kerberos ticket cache to connect to Kerberos-secured HDFS clusters.
```

Comment:
Should we link "Kerberos ticket cache" somewhere or is it something HDFS/Kerb users will definitely know already?

Possible links:
- https://arrow.apache.org/docs/python/generated/pyarrow.fs.HadoopFileSystem.html#pyarrow.fs.HadoopFileSystem
- https://web.mit.edu/kerberos/krb5-1.12/doc/basic/ccache_def.html
- https://www.hvr-software.com/docs/5/location-class-requirements/requirements-for-hdfs/hdfs-authentication-and-kerberos#HDFSAuthenticationandKerberos-AccessingKerberizedClusterswithTicketCacheFile